debian-mk-kernel.mk

```diff
@@ -13,7 +17,7 @@ all:  bbl-q bbl-u
 	@echo
 
 $(KERNEL_IMAGE): riscv-linux/.config riscv-linux/Makefile riscv-linux
-	$(MAKE) -C riscv-linux ARCH=riscv CROSS_COMPILE=riscv64-linux-gnu- vmlinux
+	$(MAKE) -C riscv-linux ARCH=riscv vmlinux
```

Comment:
Still, for clarity I'd prefer keeping `CROSS_COMPILE=$(CROSS_COMPILE)` rather than depending on passing through environment.

Reply:
Okay, I do not mind! Should I make new commit?

Reply:
Yes please!

Reply:
I made it.